documentation/docs/commands/deploying-projects.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 2
 
 # Deploy projects
 
-The tool allows for deploying a configuration or a set of configurations in the form of project(s). A project is a folder containing files that define configurations to be deployed to a environment or a group of environments. This is done by passing the `--project` flag (or `-p` for short).
+The `monaco` tool can deploy a configuration or a set of configurations in the form of project(s). A project is a folder containing files that define the configurations to be deployed to a environment or a group of environments. This is done by passing the `--project` flag (or `-p` for short).
```

Comment:
Can you please change 'monaco' to Monaco, so that it's consistent.

Reply:
Done